src/profile/main/tag/Google AdWords Remarketing - Prod_QA-UID26.js

```diff
@@ -66,11 +67,25 @@ try {
                             u.data.params[prefix] = u.data.params[prefix] || {};
                             u.data.params[prefix][e[f].substr(prefix.length + 1)] = b[d];
                         } else if (e[f].indexOf("custom.") == 0) {
+                            if (e[f].substr(7) === "isFlightConfirmation") {
+                                isFlightConfirmation = b[d];
+                            } else if (e[f].substr(7) === "bookingWindow") {
+                                bookingWindow = parseInt(b[d]);
+                            }
```

Comment:
bookingWindow is already being sent, just flightBookersMembershipTTL needs to be sent when it is a flight confirmation page. I will update the PR to not send isFlightConfirmation to google